utilities/pr_commit_counts/pr_commit_counts.py

```diff
@@ -16,27 +17,28 @@ def get_paginated_results(url, headers):
             raise Exception(f"GitHub API request failed with status code {response.status_code}")
     return results
 
-def get_repos(organization, headers):
+def get_repos(BASE_URL, organization, headers):
```

Comment:
Hmm this wasn't quite what I meant - it makes more sense to just define BASE_URL as a constant in the file as you did, and use it everywhere - you don't have to make it an argument that you pass in.